hpx/parallel/exception_list.hpp

```diff
@@ -87,11 +81,11 @@ namespace hpx { namespace parallel { inline namespace v1
                     try {
                         throw; //-V667
                     }
-                    catch(std::bad_alloc const& e) {
-                        throw e;
+                    catch(std::bad_alloc const&) {
+                        throw;
                     }
-                    catch (hpx::exception_list const& el) {
-                        throw el;
+                    catch (hpx::exception_list const&) {
+                        throw;
                     }
                     catch (...) {
                         throw hpx::exception_list(std::current_exception());
```

Comment:
Hmm, this looks really strange (it probably was strange all along). I'm sure this can be done with one level of try/catch blocks.

Reply:
@hkaiser Yes, we can do that with only one level of try/catch blocks.